lib/node_modules/@stdlib/math/base/special/hyp2f1/lib/hys2f1.js

```diff
@@ -0,0 +1,215 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var round = require( '@stdlib/math/base/special/round' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var isNonPositiveInteger = require( './utils.js' ).isNonPositiveInteger;
+
+
+// VARIABLES //
+
+var MACHEP = 1.11022302462515654042E-16;
+var EPS = 1.0e-13;
+var MAX_ITERATIONS = 10000;
+
+
+// FUNCTIONS //
+
+/**
+* Evaluates the Gaussian hypergeometric function by two-term recurrence in `a`.
+*
+* @private
+* @param {number} a - input value
+* @param {number} b - input value
+* @param {number} c - input value
+* @param {number} x - input value
+* @param {number} loss - starting loss of significance
+* @returns {Object} the function value and error
+*/
+function hyp2f1ra( a, b, c, x, loss ) {
+	var f2Val;
+	var f1Val;
+	var f0Val;
+	var err;
+	var da;
+	var f1;
+	var f0;
+	var t;
+	var n;
+
+	loss = 0.0;
+	err = 0.0;
+
+	if ( ( c < 0.0 && a <= c ) || ( c >= 0.0 && a >= c ) ) {
+		da = round( a - c );
+	}
+	else {
```

Comment:
```suggestion
	} else {
```

In general, prefer a single line, unless you need to add a comment before the `else` clause.